roles/cloud-bootstrap/tasks/main.yml

```diff
@@ -0,0 +1,5 @@
+---
+- name: Provision network resources
+  include: network.yml
+  tags:
+    - network
```

Comment:
I think, if I read the message right, this is the start of the role, and more tasks will be added to it in future PRs. This particular one just tackles the network aspect of it.